best-practices/README.md

```diff
@@ -265,6 +265,7 @@ CSS
   * Prefer `em` units instead of `px` for breakpoint values
   * Start with the smallest viewport size and work upwards using 
     `min-width`/`min-height`
+* Use double colons for pseudo-elements
```

Comment:
Can we link the W3 spec here?

Reply:
I meant the MDN doc you quoted 😆

Reply:
Good call! Added in 05c913546eca948fc2df68e9b80d88038de53ca4.